pluralize.js

```diff
@@ -476,6 +476,7 @@
     'wildebeest',
     'wildlife',
     'you',
+    '/pok(e|é)mon/',
```

Comment:
This shouldn't be working - it's a string...

Reply:
Yeah, this definitely isn't working - the tests are in the wrong place. `SINGULAR_TESTS` are for "odd plurals" to singular only (doesn't test singular to plural).

Reply:
Very good point, sorry about that!